modules/clients/pom.xml

```diff
@@ -175,6 +175,15 @@
                     <name>clientDocs</name>
                 </property>
             </activation>
+
+            <dependencies>
+                <dependency>
```

Comment:
Clients module does not depend on Ignite.NET, I think we should revert this.

Reply:
We don't need to build .NET code to generate the docs.

Reply:
Thanks.
Removed.